File: AKS_Primarility_Test/AKS.java
import java.util.Scanner;

class AKS{
    public static void main(String[] args){
    
        for(int i=0;i<1000000;i++)
        {
            if(isprm(i))
            {
                System.out.println(i);
            }
    }
    }
    
    public  static boolean isprm(long n)
    {   long i=5,w=2;
        if(n==2 || n==3)
        return true;
        if(n%2 ==0 ||n%3==0||n==1)
            return false;
            while(i*i<=n)
            {
                if(n%1==0)
                    return false;
                    i +=w;
                    w=6-w;
                    
            }
            return true;
    }
}

File: Testing/AKS_Primarility_Test/AKS.js
var isprm = function(n) {
  var i=5;
  var w=2;
  if(n==2 || n== 3)
   return 1;
  if(n%2==0 || n%3==0 || n==1)
    return 0;
  while(i*i <= n){
    if(n%i == 0)
      return 0;
    i +=w;
    w=6-w;
  }
  return 1;
}

var i=0;
for(; i<1000000; i++){
  if(isprm(i)==1)
    console.log(i + '\n');
}
